src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandName.java

```diff
@@ -0,0 +1,99 @@
+package io.stargate.sgv2.jsonapi.api.model.command;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * The canonical list of commands the API support, any command name not listed here is not a
+ * command.
+ */
+public enum CommandName {
+  // TODO: the embedding providers and offline commands should not be DML, they are not changing
+  // data
+  // they should not be DDL, they are not changing schema, we should add an CommandType.ADMIN for
+  // them ?
+
+  ALTER_TABLE("alterTable", CommandType.DDL, CommandTarget.TABLE),
+  COUNT_DOCUMENTS("countDocuments", CommandType.DML, CommandTarget.COLLECTION),
+  CREATE_COLLECTION("createCollection", CommandType.DDL, CommandTarget.KEYSPACE),
+  CREATE_INDEX("createIndex", CommandType.DDL, CommandTarget.TABLE),
+  CREATE_VECTOR_INDEX("createVectorIndex", CommandType.DDL, CommandTarget.TABLE),
+  CREATE_KEYSPACE("createKeyspace", CommandType.DDL, CommandTarget.DATABASE),
+  CREATE_NAMESPACE("createNamespace", CommandType.DDL, CommandTarget.DATABASE),
+  CREATE_TABLE("createTable", CommandType.DDL, CommandTarget.KEYSPACE),
+  DELETE_COLLECTION("deleteCollection", CommandType.DDL, CommandTarget.KEYSPACE),
+  DELETE_MANY("deleteMany", CommandType.DML, CommandTarget.TABLE, CommandTarget.COLLECTION),
+  DELETE_ONE("deleteOne", CommandType.DML, CommandTarget.TABLE, CommandTarget.COLLECTION),
+  DROP_INDEX("dropIndex", CommandType.DML, CommandTarget.KEYSPACE),
+  DROP_NAMESPACE("dropNamespace", CommandType.DDL, CommandTarget.DATABASE),
+  DROP_KEYSPACE("dropKeyspace", CommandType.DDL, CommandTarget.DATABASE),
+  DROP_TABLE("dropTable", CommandType.DDL, CommandTarget.KEYSPACE),
+  ESTIMATED_DOCUMENT_COUNT("estimatedDocumentCount", CommandType.DML, CommandTarget.COLLECTION),
+  FIND_COLLECTIONS("findCollections", CommandType.DDL, CommandTarget.KEYSPACE),
+  FIND("find", CommandType.DML, CommandTarget.TABLE, CommandTarget.COLLECTION),
+  FIND_EMBEDDING_PROVIDERS("findEmbeddingProviders", CommandType.DML, CommandTarget.DATABASE),
```

Comment:
This should be a CommandTarget.SYSTEM. As per comment on top of this class, this along with offline commands needs to to have a new CommandType.ADMIN

Reply:
Added a CommandType.ADMIN and used it for this command.